### PR TITLE
Failing tests: Fix or skip

### PR DESCRIPTION
**Pull request by jarstelfox**
qa_req 0 just passing tests

### CODE REVIEW

**Review by djmetzle — APPROVED** [review]
CR :ski:

File: frontend/tests/playwright/troubleshooting/loading.spec.ts [hunk 1]
```diff
@@ -58,11 +58,16 @@ test.describe('Vulcan Page Content and SEO', () => {
...
       ];
 
-      langsToHrefs.forEach(async (langToHref) => {
+      const checkLang = async function (langToHref: {
+         lang: string;
+         href: string;
+      }): Promise<void> {
          const hrefLang = page.locator(`link[hreflang="${langToHref.lang}"]`);
          await expect(hrefLang).toHaveAttribute('hreflang', langToHref.lang);
          await expect(hrefLang).toHaveAttribute('href', langToHref.href);
```
Comment (danielbeardsley) [inline frontend/tests/playwright/troubleshooting/loading.spec.ts:67]:
```suggestion
         return Promise.all([
            expect(hrefLang).toHaveAttribute('hreflang', langToHref.lang),
            expect(hrefLang).toHaveAttribute('href', langToHref.href)
         ]);
```

Might as well run those in parallel too.

**Comment by danielbeardsley** [comment]
Good job finding this @jarstelfox !

**Review by danielbeardsley — APPROVED** [review]
CR :+1:

**Pull request merged**